prowler/providers/aws/services/cognito/cognito_service.py

```diff
@@ -119,4 +340,25 @@ class UserPool(BaseModel):
     status: str
     password_policy: Optional[dict]
     mfa_config: Optional[MFAConfig]
-    tags: Optional[list] = []
+    tags: Optional[dict]
+    account_recovery_settings: Optional[dict]
+    user_pool_client: Optional[UserPoolClient]
```

Comment:
Please, create a list of UserPoolClient since there can be more than one.

Reply:
Done